Merge the WCA Regulations and Guidelines (#1287)
thewca/wca-regulations: wca-regulations.md

# <wca-title>WCA Regulations

> <version>Version: July 16, 2025

Author: ```suggestion
<version>Version: July 17, 2025
```

Please note that translations are not official versions, even though they are provided on the WCA website for convenience.
If there is a difference between a translation and the (current official) English version, the English version must be used.

### WCA Regulations

The WCA Regulations contain the full set of Regulations that apply to all official competitions sanctioned by the World Cube Association.

> Previously, from the January 1st, 2013 release until the January 1st, 2025 release, the WCA Regulations were split into the WCA Regulations and the WCA Guidelines. The former contents of both documents were combined into the WCA Regulations for the DATE, 2025 release.

Author: ```suggestion
Previously, from the January 1st, 2013 release until the January 1st, 2025 release, the WCA Regulations were split into the WCA Regulations and the WCA Guidelines. The former contents of both documents were combined into the WCA Regulations for the July 17th, 2025 release.
```

        - 5b5f+) [EXAMPLE] There is a [Visual Guide](https://drive.google.com/file/d/15XszaCGNvy3Dk6X6qERzZWZaDH1RH04z/view) available for [Regulation 5b5f](regulations:regulation:5b5f).
        - 5b5f++) [EXAMPLE] A 4x4x4 Cube edge "wing" twisted on its spot is considered solved.
        - 5b5f+++) [EXAMPLE] A corner affected by a corner twist is considered to be placed in a valid position, but such position does not render the piece solved.
        - 5b5f++++) [CLARIFICATION] If there are multiple pieces partially detached or not fully placed, their final position is assessed one at a time. The piece that is closest to an open position is assessed first, and is then considered to occupy that position. This process is repeated until all partially detached or not fully placed pieces have been assessed.
    - 5b6) Definition: A part is one or more fully connected components from a group of puzzle components that remain connected and do not move relative to each other during ordinary operation of the puzzle (e.g. a 3x3x3 center cap, a Pyraminx tip, a 3x3x3 corner piece missing one face, one face of a 3x3x3 corner piece).

> - 5c+) [REMINDER] If a competitor has a puzzle defect, this does not grant them the right to an extra attempt.   

Reviewer: ```suggestion
- 5c+) [REMINDER] If a competitor has a puzzle defect, this does not grant them the right to an extra attempt.
```

        - B4c1) In all cases, the competitor must wear the blindfold such that their view of the puzzle would still clearly be blocked if the sight blocker were not in the way.
        - B4c2) A sight blocker must be positioned such that the competitor's view of the puzzle would still clearly be blocked if they were not wearing a blindfold.
        - B4c3) If the judge and competitor agree beforehand, or if required by the organization team, a suitable object (e.g. a music stand, the surface of the table) placed by the competitor may be used as the sight blocker. The competitor must make a reasonable attempt to place the sight blocker themselves. Penalty: disqualification of the attempt (DNF). If the sight blocker is not positioned such that the competitor's view of the puzzle would still clearly be blocked if they were not wearing a blindfold, an extra sight blocker should be placed by the judge.
    - B4d) The competitor must not look at the puzzle at any point during the blindfolded phase. Penalty: disqualification of the attempt (DNF).

>     - B4d+) [CLARIFICATION] For 3x3x3 Blindfolded, competitors are permitted to determine the location of a permitted logo (see [Regulation 3l](regulations:regulation:3l)) by touch during the blindfolded phase. 

Reviewer: ```suggestion
    - B4d+) [CLARIFICATION] For 3x3x3 Blindfolded, competitors are permitted to determine the location of a permitted logo (see [Regulation 3l](regulations:regulation:3l)) by touch during the blindfolded phase.
```
